5630 add documentation for recovery device feature (#5655)
Scille/parsec-cloud: docs/userguide/recovery_device.rst

   .. image:: screens/recovery_device_menu.png
      :alt: Parsec menu with recovery device option highlighted

2. That will open a modal that look like this:

   .. image:: screens/recovery_device_modal.png
      :alt: Parsec recovery device modal

Create a new recovery device
############################

1. At that point, you select ``Create a recovery device``, that will open a new modal:

   .. image:: screens/recovery_device_create_device.png
      :alt: Create a recovery device modal

2. From here:

   a. You select the device you would like to back up.
   b. The file where the recovery device would be located.

3. Click next, you will be prompted to enter the password for the device you have selected

   .. image:: screens/recovery_device_create_device_re_auth.png
      :alt: Create a recovery device re-authenticate modal

4. After confirmation, you are at the last step. Here you need to save the displayed passphrase

   .. image:: screens/recovery_device_create_device_confirmation.png
      :alt: Create a recovery device confirmation modal

   .. note::

>       The passphrase should be handled like any password.
> 
>       You could print out the passphrase and put-it in a safe to secure it.
>       Nevertheless, you should not save it alongside the recovery device.

Reviewer: ```suggestion
      The passphrase should be handled carefully like any other password.

      You can print out the passphrase and put-it in a safe to secure it.
      Nevertheless, you should not save it alongside the recovery file.
```

Use a recovery device
#####################

> 1. At that point, you select ``Recover a Device``, that will open a new modal:

Reviewer: ```suggestion
1. In the Device recovery dialog, select ``Recover a Device``
```

>    .. image:: screens/recovery_device_recover.png
>       :alt: Recover a device modal
> 
> 2. From here:
> 
>    a. You select the recovery device's file that correspond to the device you want to recover.
>    b. Enter the passphrase for the recovery device.
>    c. Enter the desired device name.

Reviewer: ```suggestion
2. From here:

   a. Select the recovery file of the device you want to recover.
   b. Enter the passphrase for the recovery device.
   c. Enter a new name for the device.

   .. image:: screens/recovery_device_recover.png
      :alt: Recover a device modal
```

3. Click next, you will be prompted to enter a new password and confirm it.

   .. image:: screens/recovery_device_recover_new_password.png
      :alt: Enter the new password for the recovered device modal

> 4. The last modal is a confirmation prompt indicating everything went fine, you can click on continue.

Reviewer: ```suggestion
4. A message is displayed to confirm the device has been recovered, you can click `Continue`.
```

   .. image:: screens/recovery_device_recover_confirmation.png
      :alt: Device recovered confirmation modal

> 5. You will find the recovered device in the devices' list

Reviewer: ```suggestion
5. You can now access Parsec as usual using the new recovered device from the devices' list
```
